docs/scripts.md

```diff
@@ -1,26 +1,66 @@
 # Truffle Scripts
```

Comment:
shouldn't the filename would be `docs/truffle-scripts.md` just to be consistent with the other docs

Reply:
Done